buildscripts/cstor-csi-driver/Dockerfile

```diff
@@ -1,25 +1,39 @@
+# Copyright 2020 The OpenEBS Authors. All rights reserved.
 #
-# This Dockerfile builds a recent volume-mgmt using the latest binary from
-# volume-mgmt  releases.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 
 FROM ubuntu:16.04
 RUN apt-get update; exit 0
 RUN apt-get -y install rsyslog xfsprogs
 #RUN apt-get clean && rm -rf /var/lib/apt/lists/*
 
+ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL
+ARG DBUILD_SITE_URL
```

Comment:
If we are using these args here, why are we passing ${DBUILD_ARGS} from Makefile 
${DBUILD_ARGS} also contains the same arguments.

Reply:
Specifying as ARG ensures that docker build contains the required build args. If a ENV is not set, then docker build command will fail. 

Alternate way is to directly use ENV in the LABEL, and assume that ENVs are set.

Reply:
ARG is like build-time variables. They are only available from the moment they are 'announced' in the Dockerfile with an ARG instruction till the image is built.
more info https://docs.docker.com/engine/reference/builder/#arg